platform/tron/api.go

```diff
@@ -64,20 +95,56 @@ func Normalize(srcTx *Tx) (tx blockatlas.Tx, ok bool) {
 			ID:   srcTx.ID,
 			Coin: coin.TRX,
 			Date: srcTx.BlockTime / 1000,
-			From: from,
-			To:   to,
 			Fee:  "0",
+			From: from,
+			To: to,
 			Meta: blockatlas.Transfer{
 				Value:    transfer.Value.Amount,
 				Symbol:   coin.Coins[coin.TRX].Symbol,
 				Decimals: coin.Coins[coin.TRX].Decimals,
 			},
 		}, true
+	}
+
+	return tx, false
+}
+
+func NormalizeTokenTransfer(srcTx *Tx, tokenInfo AssetInfo) (tx blockatlas.Tx, ok bool) {
+	contract := &srcTx.Data.Contracts[0]
+
+	switch contract.Parameter.(type) {
+	case TransferAssetContract:
+		transfer := contract.Parameter.(TransferAssetContract)
+		from, err := HexToAddress(transfer.Value.OwnerAddress)
+		if err != nil {
+			return tx, false
```

Comment:
return the error instead false, we need to track all errors

Reply:
https://github.com/trustwallet/blockatlas/pull/334/files#diff-5ce8d7a827e39cccb683f35e56f3c502R95